src/sentry/static/sentry/app/stores/tooltipStore.tsx

```diff
@@ -16,9 +16,11 @@ const TooltipStore: TooltipStoreInterface = {
   getOpenableSingleTooltips() {
     return this.tooltips.filter(tooltip => {
       // Filtering out disabled tooltips and lists of tooltips (which cause rendering issues for snapshots) using the internal 'key'
+      const internals =
```

Comment:
Maybe `_reactInternals` or `_internals` so that people don't try to re-use it?

Reply:
Good call